apps/cli-daemon/src/app/workspace/dto/workspace-create.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { IsArray, IsNotEmpty, IsOptional, IsString } from 'class-validator';
+
+export class WorkspaceCreateDto {
+  @IsNotEmpty()
+  @IsString()
```

Comment:
Maybe add error message `path is mandatory` or something

Reply:
Don't you prefer the build in nest js message of the IsNotEmpty decorator? 

`
{
    "statusCode": 400,
    "message": [
        "path should not be empty"
    ],
    "error": "Bad Request"
}
`